eng/scripts/Manage-SDKSuppression-Label.ps1

```diff
@@ -0,0 +1,386 @@
+Install-Module -Name powershell-yaml
+
+. $PSScriptRoot/../common/scripts/common.ps1
+
+$baseOwner = $env:GITHUB_EVENT_PULL_REQUEST_BASE_REPO_OWNER
+$baseRepo = $env:GITHUB_EVENT_PULL_REQUEST_BASE_REPO_NAME
+$baseBranch = $env:GITHUB_EVENT_PULL_REQUEST_BASE_REF
+
+$headOwner = $env:GITHUB_EVENT_PULL_REQUEST_HEAD_REPO_OWNER
+$headRepo = $env:GITHUB_EVENT_PULL_REQUEST_HEAD_REPO_NAME
+$headBranch = $env:GITHUB_EVENT_PULL_REQUEST_HEAD_REF
+
+$prNumber = $env:GITHUB_EVENT_PULL_REQUEST_NUMBER
+
+$GithubToken = $env:TOKEN
+
+Write-Host "Base Owner: $baseOwner"
+Write-Host "Base Repo: $baseRepo"
+Write-Host "Base Branch: $baseBranch"
+Write-Host "Head Owner: $headOwner"
+Write-Host "Head Repo: $headRepo"
+Write-Host "Head Branch: $headBranch"
+Write-Host "Pull Request Number: $prNumber"
+
+$SuppressionFileName = "sdk-suppressions.yaml"
+
+$sdkLabels = @{
+    "azure-cli-extensions"           = @{
```

Comment:
Both 'azure-cli-extensions' and 'azure-resource-manager-schemas' can be deleted as we won't support them.